tap_hubspot/hubspot.py

```diff
@@ -12,6 +12,8 @@
 
 LOGGER = singer.get_logger()
 hs_calculated_form_submissions = []
+event_contact_ids = []
```

Comment:
Store it as an attribute on Hubspot class instead of creating globals 👍 

https://stackoverflow.com/questions/19158339/why-are-global-variables-evil

Reply:
^ link for reference on the topic